plugins/github-pull-requests-board/src/api/useGetPullRequestsFromUser.ts

```diff
@@ -0,0 +1,110 @@
+/*
+ * Copyright 2022 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import React from 'react';
+
+import {
+  GraphQlUserPullRequests,
+  PullRequestsNumberAndOwner,
+} from '../utils/types';
+import { useOctokitGraphQl } from './useOctokitGraphQl';
+
+const PULL_REQUEST_LIMIT = 10;
+const GITHUB_GRAPHQL_MAX_ITEMS = 100;
+
+export const useGetPullRequestsFromUser = () => {
+  const graphql =
+    useOctokitGraphQl<GraphQlUserPullRequests<PullRequestsNumberAndOwner[]>>();
+
+  const fn = React.useRef(
```

Comment:
Interesting. I'd say this should be using `useCallback`, instinctively

Reply:
Not super familiar with React and basically following the way things have been implemented under `src/api` (i.e.: `useGetPullRequestDetails`, `useGetPullRequestsFromRepository`).

I double-checked, and it seems like these functions are all called from within a `useCallback` in `usePullRequestsByTeam`, so I guess it's having the memoization properties that you're looking for already?